src/app/components/LeftBanner.tsx

```diff
@@ -0,0 +1,45 @@
+import style from './LeftBanner.module.css';
+import { useStepsContext, STEPS_NAMES } from '../context/stepsContext';
+
+export function LeftBanner() {
+    const value = useStepsContext()
```

Comment:
variable naming is unclear
```suggestion
    const stepContext = useStepsContext()
```